sdk/appconfiguration/azure-data-appconfiguration/src/main/java/module-info.java

```diff
@@ -3,6 +3,8 @@
 
 module com.azure.data.appconfiguration {
     requires transitive com.azure.core;
+    requires transitive com.azure.http.netty;
+    requires java.sql;
```

Comment:
Why do we need java.sql?

Reply:
`java.sql` is required to compile samples. I also agree that this isn't the right location for this and should instead investigate making the test `module-info` that was removed in the PR work better with IDEs like IntelliJ

Reply:
will revert the changes